lib/posthog/feature_flags.rb

```diff
@@ -76,6 +75,21 @@ def load_feature_flags(force_reload = false)
       end
     end
 
+    def get_feature_variants(distinct_id)
+      request_data = {
+            "distinct_id": distinct_id,
+            "personal_api_key": @personal_api_key,
```

Comment:
are groups required for this?

Reply:
yep, groups support multivariate flags too. So the signature is exactly the same as `is_feature_enabled`

Reply:
OKKK I ADDED ITTT